docs/design/rescheduling.md

```diff
@@ -0,0 +1,133 @@
+### Rescheduling Strategy
+* OfflineOnly
+
+    `OfflineOnly`, abbreviated as `OLO`, will only pick out offline workloads, which are attached with annotation
+`preemptable: true`, and then reschedule them only.
+
+* LowPriorityFirst
+
+    `LowPriorityFirst`, abbreviated as `LPF`, will sort workloads by priority and then reschedule pods in ascending 
+order.
+
+* ShortLifeTimeFirst
+
+  `ShortLifeTimeFirst`, abbreviated as `SLTF`, will sort workloads by running time. Pods with the shortest lift time 
+will be rescheduled first. This strategy can make sure workloads with long task type goes healthily without interruptions.
+
+* BigObjectFirst
+
+    `BigObjectFirst`, abbreviated as `BOF`, will select workloads which request the most **dominate resource** and reschedule
+them first. It helps improve system throughout and avoid small workloads' starvation.
+
+* MoreReplicasFirst
+  
+    `MoreReplicasFirst`, abbreviated as `MRF`, will sort workloads by replicas number. Workloads with the most replicas
+will be rescheduled first. This strategy is friendly to `gang scheduling` for it will consider the `minAvailable` in 
+volcano jobs.
+
+### Metrics
+All the decisions made by rescheduling strategies will consider the metrics from `Prometheus`. Namely, Volcano will 
+list the real node resource utilization and pod distribution instead of request resource. Basically, usage of`CPU` and 
+`Memroy` will be collected. Other resource such as `GPU` can be extended.
+
+## Implementation(To Be Discussed)
+```yaml
+## Configuration Option 1
+actions: "enqueue, allocate, backfill, shuffle"  ## add 'shuffle' at the end of the actions
+action_configurations:       ## configurations for actions
+  - name: shuffle
+    arguments:
+      interval: 5m           ## optional, the frequency how long run rescheduling strategies once. 5 minutes by default.
+      queueSelector:         ## optional, select workloads in specified queues as potential evictees. All queues by default.
+        - default
+        - test-queue
+      labelSelector:         ## optional, select workloads with specified labels as potential evictees. All labels by default.
+        business: offline
+        team: test
+tiers:
+  - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+      - name: rescheduling       ## rescheduling plugin
+        arguments:        
+          offlineOnly: true
+          lowPriorityFirst: true
+  - plugins:
+      - name: overcommit
+      - name: drf
+      - name: predicates
+      - name: proportion
+      - name: nodeorder
+      - name: binpack
+```
+```yaml
+## Configuration Option 2
+actions: "enqueue, allocate, backfill"
+tiers:
+  - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+  - plugins:
+      - name: overcommit
+      - name: drf
+      - name: predicates
+      - name: proportion
+      - name: nodeorder
+      - name: binpack  
+rescheduling:            ## configurations for rescheduling
```

Comment:
what is `rescheduling`, action or plugin?

Reply:
> what is `rescheduling`, action or plugin?

Neithor is action or plugins. In fact, it works for 'shuffle' action. Unfortunately, there is no configuration options for actions. So I have to add these configurations.